ctx.go

```diff
@@ -380,37 +362,32 @@ func (ctx *Ctx) Format(body interface{}) {
 	// Format based on the accept content type
 	switch accept {
 	case "html":
-		ctx.SendString("<p>" + b + "</p>")
+		return c.SendString("<p>" + b + "</p>")
 	case "json":
-		if err := ctx.JSON(body); err != nil {
-			ctx.Send(body) // Fallback
-			log.Println("Format: error serializing json ", err)
-		}
+		return c.JSON(body)
 	case "txt":
-		ctx.SendString(b)
+		return c.SendString(b)
 	case "xml":
 		raw, err := xml.Marshal(body)
 		if err != nil {
-			ctx.Send(body) // Fallback
-			log.Println("Format: error serializing xml ", err)
-		} else {
-			ctx.Fasthttp.Response.SetBodyRaw(raw)
+			return fmt.Errorf("error serializing xml: %v", body)
 		}
-	default:
-		ctx.SendString(b)
+		c.fasthttp.Response.SetBody(raw)
+		return nil
 	}
+	return c.SendString(b)
 }
 
 // FormFile returns the first file by key from a MultipartForm.
-func (ctx *Ctx) FormFile(key string) (*multipart.FileHeader, error) {
-	return ctx.Fasthttp.FormFile(key)
+func (c *Ctx) FormFile(key string) (*multipart.FileHeader, error) {
+	return c.fasthttp.FormFile(key)
 }
 
 // FormValue returns the first value by key from a MultipartForm.
 // Returned value is only valid within the handler. Do not store any references.
 // Make copies or use the Immutable setting instead.
-func (ctx *Ctx) FormValue(key string) (value string) {
-	return getString(ctx.Fasthttp.FormValue(key))
+func (c *Ctx) FormValue(key string, defaultValue ...string) string {
```

Comment:
This new `defaultValue` should be documented.

Reply:
Done